### DISCUSSION TITLE
VS Code command "code" does not persist in ZSH Terminal

### DISCUSSION THREAD

**Discussion by JordanMcgrew** [ohmyzsh/ohmyzsh#11124] [Help] [answered]
I genuinely have tried everything I can think of and it seems the computer gods are having a laugh at my expense. If I restart my computer or quit the terminal, its as if the code command was never installed and I have to uninstall/reinstall the code command. 
I've tried manually removing the code file from usr/local/bin and reinstalling it, didn't work. I've reinstall VS Code, nope. Pretty confused, I'm assuming its got to be a command pathing issue perhaps but I'm curious if anyone else has experienced this.

**Comment by HwangTaehyun** [comment, marked as answer]
Hi! @JordanMcgrew 
You should add this line to ~/.zshrc
```
code () { VSCODE_CWD="$PWD" open -n -b "com.microsoft.VSCode" --args $* ;}
```

Reload your ~/.zshrc, then ```code .``` in your project. It works!
